core/controllers/base.py

```diff
@@ -238,8 +238,8 @@ def dispatch(self):
                         'Registration session expired.')
                 csrf_token = self.request.get('csrf_token')
                 if not csrf_token:
-                    raise Exception(
-                        'Missing CSRF token. Changes were not saved. '
+                    raise self.UnauthorizedUserException(
+                        'Missing CSRF token. Changes were not saved,'
```

Comment:
Use a period, like before; add a space at the end.

Reply:
done